docs/participate/collator/3_setup_node.md

```diff
@@ -0,0 +1,167 @@
+
+:::
+
+### Configuration
+
+Running a collator requires a few configuration parameters.
+Some of the parameters might appear twice in the command to start the collator, the reason being that a parachain collator actually runs two blockchains.
+The parameter that are listed before the `--` are related to the parachain node itself (the KILT blockchain), whereas the parameters following the `--` are related to the relaychain, e.g., Kusama or Polkadot.
+
+Following is a description of some of the parameters that can be set when spinning up a parachain collator node.
+
+#### RPC and WS Endpoints
+
+The collator needs session keys that connect it with the collator's KILT account.
+These session keys can be generated by calling an RPC endpoint that the collator optionally exposes.
+Exposing the RPC endpoint can be done using the following parameters:
+
+```
+--rpc-port=9933
+--rpc-cors=all
+--rpc-methods=unsafe
+```
+
+Exposing the RPC endpoint of a collator does not imply that it becomes accessible via the PolkadotJS Apps interface, because this requires a Websocket to connect to the node.
+
+By default, the Websocket port used by the node is configured to be `9944`, but it can be changed by specifying a different value with `--ws-port=<ws_port>`.
+
+Connecting from a remote host to either the collator RPC endpoint or WS endpoint requires to explicitly expose those endpoints to the public with the `--rpc-external` and `--ws-external` options.
+
+:::danger
+
+Be aware that it is highly discouraged to publicly expose an RPC endpoint, especially if it allows the execution of unsafe RPC calls!
+You should be the only one able to call the RPC endpoint.
+For a secure setup, follow the instructions in the section about [generating the session keys](#generating-session-keys).
+
+:::
+
+#### WASM runtime execution
+
+A KILT collator should use the `--execution=wasm` parameter for both the relaychain and parachain collation.
+The alternative to WASM runtime execution is native runtime execution, which might be faster but can, in some cases, deviate from the WASM execution logic and result in a different state.
+When this happens, the collator will crash and will stop producing blocks.
+Since the WASM runtime logic is part of the blockchain state itself and hence represents the single source of truth, all collators should execute the WASM version of the runtime logic.
+
+#### Specify the right chain spec
+
+The `--chain` parameter decides which blockchain the KILT collator will join.
+This parameter must be specified for both the parachain and the relaychain, since both chains are, as a matter of fact, separate blockchains.
+The KILT parachain accepts an additional parameter to select the environment to use for the WASM runtime execution.
+This can either be `peregrine` or `spiritnet`.
+
+Hence, to start a collator for the Spiritnet network, the parameter would be `--chain=spiritnet`, while for Peregrine would be `--chain=peregrine`.
```

Comment:
Right now, `--chain=peregrine` does not work, as there's no hardcoded spec for that.

Reply:
Good catch. This was copied over from the existing guide and also lived in the full node tutorial. Fixed in [4b99410](https://github.com/KILTprotocol/docs/pull/72/commits/4b9941079def21d6ea51c2d7df7ea315677f6366).